.github/workflows/test.yml

```diff
@@ -35,7 +35,6 @@ jobs:
 
     - name: 🔧 - Install Dependencies
       run: |
-        pip install -U git+https://github.com/EBI-Metagenomics/ena-api-handler.git
```

Comment:
Nice! Also need to remove this line from the repo README and from the Dockerfiles in `docker/`.

Reply:
I'll do it in a bit then :)